dts/bindings/ethernet/atmel,gmac-common.yaml

```diff
@@ -56,3 +56,9 @@ properties:
   mac-eeprom:
     type: phandle
     description: phandle to I2C eeprom device node.
+
+  phy-dev:
+    required: true
+    type: phandle
+    description: |
+      Ethernet PHY device managed by this network interface.
```

Comment:
This should be at ethernet-controller.yaml because it is a common property for all ethernet controllers.

Reply:
Good idea: https://github.com/zephyrproject-rtos/zephyr/pull/60889